samples/sensor/lis3dh_test/README.rst

```diff
@@ -0,0 +1,75 @@
+.. _lis2dh:
+
+LIS2DH: Motion Sensor Monitor
+#############################
+
+Overview
+********
+
+This sample application periodically reads accelerometer data from the
+LIS2DH sensor (or the compatible LS2DH12, LIS3DH, and LSM303DLHC
+sensors), and displays the sensor data on the console.
+
+Requirements
+************
+
+This sample uses the LIS2DH, ST MEMS system-in-package featuring a 3D
```

Comment:
specify more specific requirements (only lis2dh, or also other version like lis3dh; certain microcontroller that has specific peripheral pins? etc)

Reply:
I haven't touched this file yet, it's currently an exact copy of the README from the existing LIS2DH sample application.

Reply:
Thought so :), then see this as a reminder to not forget it ;)